ext/SparseConnectivityTracerNNlibExt.jl

```diff
@@ -0,0 +1,97 @@
+# NNlib activation functions on tracers.
+# Parametric activation functions with two or more arguments are ignored.
+module SparseConnectivityTracerNNlibExt
+
+if isdefined(Base, :get_extension)
+    import SparseConnectivityTracer as SCT
+    using NNlib
+else
+    import ..SparseConnectivityTracer as SCT
+    using ..NNlib
+end
+
+## 1-to-1
+
+# ops_1_to_1_s: 
+# x -> f  != 0
+# ∂f/∂x   != 0
+# ∂²f/∂x² != 0
+ops_1_to_1_s = (
+    # ReLU-like activation functions
+    celu,
+    elu,
+    gelu,
+    selu,
+    # Other activation functions
+    σ, # sigmoid
+    hardswish,
+    lisht,
+    logσ,
+    logcosh,
+    mish,
+    sigmoid_fast,
+    softplus,
+    softsign,
+    swish,
+    tanh_fast,
+    tanhshrink,
+)
+
+for op in ops_1_to_1_s
+    T = typeof(op)
+    @eval SCT.is_influence_zero_global(::$T) = false
+    @eval SCT.is_firstder_zero_global(::$T) = false
+    @eval SCT.is_seconder_zero_global(::$T) = false
+end
+
+SCT.is_seconder_zero_local(::typeof(celu)) = x > 0
+SCT.is_seconder_zero_local(::typeof(elu)) = x > 0
+SCT.is_seconder_zero_local(::typeof(selu)) = x > 0
+
+SCT.is_firstder_zero_local(::typeof(hardswish)) = x < -3
```

Comment:
the first derivative is never zero?

Reply:
`hardswish(x) = x * hardσ(x)` and 
`hardσ(x) = max(0, min(1, (x + 3) / 6))`, so we get a function that's zero until -3, quadratic until 3 and linear afterwards:

https://www.wolframalpha.com/input?i=x+*+max%280%2C+min%281%2C+%28x+%2B+3%29+%2F+6%29%29

Reply:
right, my bad